h/static/scripts/hypothesis.js

```diff
@@ -1,3 +1,8 @@
+if (window.annotator) {
+    console.warn("Not deploying Hypothes.is, in order to avoid double deployment.");
```

Comment:
We also just got rejected from the Mozilla add-on review for logging messages to the console.